simple-todo/webpack.config.js

```diff
@@ -9,33 +15,29 @@ module.exports = {
     filename: 'build.js'
   },
   module: {
-    loaders: [
-      {
-        test: /\.vue$/,
-        loader: 'vue'
-      },
-      {
-        test: /\.js$/,
-        loader: 'babel',
-        exclude: /node_modules/
-      },
+    rules: [
+      { test: /\.vue$/, loader: 'vue-loader' },
+      { test: /\.css$/, loader: 'css-loader' },
```

Comment:
`npm run dev` throws an error if this is missing.
I don't believe the SCSS variant is needed, but it adds support for <style lang="scss" /> in vue and scss formatted file extensions. There is a citation in the documentation, but I don't have it handy.